src/docs/product/accounts/sso/ping-sso.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "Ping Identity SSO"
+sidebar_order: 3
+description: "Set up Ping Identity Single Sign-On (SSO) on Sentry."
+---
+
+## Installation
+
+1. Log in to [Ping Identity](https://www.pingidentity.com/en/account/sign-on.html).
+
+1. Go to "Adminstrators" environment for your organization.
+
+1. If you don't require a permission group for Sentry, skip to Step 5.
+
+1. In the sidebar, click on "Identities" then "Groups", and navigate to it. From there, create a new group, add yourself and other members to it. For more details about group creation, see the [Ping docs](https://docs.pingidentity.com/bundle/pingone/page/vdz1610472224361.html).
+
+1. In the sidebar, click on "Connections" then "Applications". Click on "+ Application" and for the application type, select "Web App" and "SAML".
+
+   ![Create web application for Ping](ping-new-app.png)
+
+1. For Section (1), labeled "Create App Profile", you can get the Sentry logo from [here](https://sentry.io/branding/).
+
+1. For Section (2), labeled "Configure SAML", select "Manually Enter". Enter the following data in each line and save your changes.
+
+   - Assertion Consumer Service (ACS) URL: `https://sentry.io/saml/acs/YOUR_ORG_SLUG/`
+
+   - Entity ID: `https://sentry.io/saml/metadata/YOUR_ORG_SLUG/`
+
+   - Single-Logout (SLO) Endpoint: `https://sentry.io/saml/sls/YOUR_ORG_SLUG/`
+
+   - Target Application URL: `https://sentry.io/organizations/YOUR_ORG_SLUG/`
+
+1. For Section (3), labeled "Attribute Mappings". For "Application Attribute" ← "Outgoing Value", use:
```

Comment:
should that be a left arrow?

```suggestion
1. In Section (3), labeled "Attribute Mappings", for "Application Attribute" ← "Outgoing Value", use:
```

Reply:
Yea, that's the direction of the arrow in Ping's UI.